src/platform-includes/getting-started-install/kotlin-multiplatform.mdx

```diff
@@ -0,0 +1,48 @@
+### Prerequisites (Android)
+
+Android require disabling auto-init on Android to not clash with the ContentProvider, which auto-initializes the Sentry Android SDK. To do so, add the following to the AndroidManifest.xml file under your androidMain source set:
+
+```kotlin
+<application>
+    <meta-data android:name="io.sentry.auto-init" android:value="false" />
+</application>
+```
+
+To install the Kotlin Multiplatform SDK, you need to add the following to your `build.gradle.kts` file in your shared module:
```

Comment:
Usually, there's multiple build.gradle.kts files, is this the root one or the app/build.gradle.kts one? let's be specific here.

Reply:
`shared module` should be the identifying folder here. When you create a brand new KMP app on Android studio or Intellij it will create a subproject `shared`, that's where the build.gradle is

Reply:
so maybe the file name should be shared/build.gradle.kts?

Reply:
yeah thats a good idea